.github/release-drafter.yml

```diff
@@ -20,4 +20,4 @@
 _extends: maven-gh-actions-shared
 version-template: '$MAJOR'
 name-template: '$NEXT_MAJOR_VERSION'
-tag-template: 'maven-parent-$NEXT_MAJOR_VERSION'
+tag-template: 'v$NEXT_MAJOR_VERSION'
```

Comment:
What about minor and bugfix versions?

Reply:
Parents project has only one number in version ... 
https://github.com/apache/maven-parent/releases

Reply:
> Parents project has only one number in version ... https://github.com/apache/maven-parent/releases

Ah thought this gets inherited to childs, thanks for info.